task-list/app/views/people/index.html.erb

```diff
@@ -0,0 +1,22 @@
+<table>
+  <col width="80%">
```

Comment:
It might make more sense to have a class associated with these columns that will then set the width